src/Store.js

```diff
@@ -1,13 +1,35 @@
-module.exports = {
-  getInitialState() {
-    return undefined
+/**
+ * Store
+ * Used to provide default values for a store configuration
+ */
+
+function Store (config, id) {
+  // Fold configuration over self
+  for (var i in config) {
+    this[i] = config[i]
+  }
+
+  this.toString = () => id
+}
+
+Store.prototype = {
+  getInitialState() {},
+
+  willRespondTo(action) {
+    return action in this
+  },
+
+  transform(state, action, params) {
+    return this.willRespondTo(action) ? this[action](state, params) : state
   },
 
   serialize(state) {
     return state
   },
 
-  deserialize(state = this.getInitialState()) {
+  deserialize(state) {
     return state
```

Comment:
Ya. It's sort of like how `parse` works in Backbone. By default, I don't want to apply any parsing at all, just pass the value through. By defining it here, I don't have to make a bunch of checks for if `deserialize` is defined.